app/forms.py

```diff
@@ -1,55 +1,49 @@
+from flask_login import current_user
 from flask_wtf import FlaskForm
+from flask_wtf.file import FileAllowed
 from wtforms import StringField, PasswordField, BooleanField, SubmitField, TextAreaField, MultipleFileField, SelectField, FileField
 from wtforms.validators import ValidationError, DataRequired, Email, EqualTo, Length
-from flask_login import current_user
 
-from flask_wtf.file import FileAllowed
 from app.models import User, Topic
 
-class LoginForm(FlaskForm):
-    email = StringField('Email', validators=[DataRequired()])
+class EmailFormMixin():
+    email = StringField('Email', validators=[DataRequired(), Email()])
+
+class LoginForm(FlaskForm, EmailFormMixin):
     password = PasswordField('Password', validators=[DataRequired()])
     remember_me = BooleanField('Remember Me')
     submit = SubmitField('SIGN IN')
 
-class DeactivateForm(FlaskForm):
-    email = StringField('Email', validators=[DataRequired()])
+class DeactivateForm(FlaskForm, EmailFormMixin):
     submit = SubmitField('DEACTIVATE')
 
-class RegistrationForm(FlaskForm):
+class RegistrationForm(FlaskForm, EmailFormMixin):
     firstName = StringField('First Name', validators=[DataRequired()])
     lastName = StringField('Last Name', validators=[DataRequired()])
-    email = StringField('Email', validators=[DataRequired(), Email()])
     password = PasswordField('Password', validators=[DataRequired()])
     password2 = PasswordField(
         'Repeat Password', validators=[DataRequired(), EqualTo('password')])
     submit = SubmitField('Register')
 
-    #def validate_username(self, username):
-    #    user = User.query.filter_by(username=username.data).first()
-    #    if user is not None:
-    #        raise ValidationError('Please use a different username.')
-
     def validate_email(self, email):
         user = User.query.filter_by(email=email.data).first()
         if user is not None:
             raise ValidationError('Please use a different email address.')
 
-class ContactForm(FlaskForm):
+class ContactForm(FlaskForm, EmailFormMixin):
     name = StringField('Name', validators=[DataRequired()])
```

Comment:
If we are going to use Mixin, we could also use the StringFormMixin for the string fields like name as well right?

Reply:
I suppose. Not important for now. Would prevent the use of `form.label` if wanted.

Reply:
Yea, I approved it. Maybe can think about it in Milestone 3 xD